bin/pygrb/pycbc_pygrb_plot_injs_results

```diff
@@ -283,156 +255,113 @@ if not os.path.isdir(outdir):
-# quantity is not defined for *missed* injections
-if 'sky_error' in [x_qty, y_qty]:
-    found_inj['ra'] = np.asarray(found_injs.get_column('longitude'))
-    found_inj['dec'] = np.asarray(found_injs.get_column('latitude'))
-    found_trig['ra'] = np.asarray(found_trigs.get_column('ra'))
-    found_trig['dec'] = np.asarray(found_trigs.get_column('dec'))
-    found_inj['sky_error'] = np.arccos(np.cos(found_inj['dec'] - found_trig['dec']) -\
-                                       np.cos(found_inj['dec']) * np.cos(found_trig['dec']) *\
-                                       (1 - np.cos(found_inj['ra'] - found_trig['ra'])))
-    missed_inj['sky_error'] = np.array([])
-
-# Construct conditions for injection:
-# 1) found louder than background,
-zero_fap = np.zeros(len(found_injs)).astype(bool)
-zero_fap_cut = found_trig['bestnr'] > max_bestnr
-zero_fap = zero_fap | (zero_fap_cut)
-# 2) found (bestnr > 0) but not louder than background (non-zero FAP)
-nonzero_fap = ~zero_fap & (found_trig['bestnr'] != 0)
-# 3) missed after being recovered (i.e., vetoed)
-missed = (~zero_fap) & (~nonzero_fap)
+found_inj = {x_qty: inj_data[x_qty][found],
+             y_qty: inj_data[y_qty][found]}
+
+# Extract the detection statistic of injections found after vetoes
+found_after_vetoes_stat = f['found_after_vetoes/stat'][:]
 
 # Separate triggers into:
-# 1) Zero FAP (zero_fap) 'g_found'
-g_found = grab_triggers_subset(found_inj, zero_fap, found_trig)
-# 2) Non-zero FAP (nonzero_fap) 'g_ifar'
-g_ifar = grab_triggers_subset(found_inj, nonzero_fap, found_trig)
-g_ifar['stat'] = np.zeros([len(g_ifar['bestnr'])])
-for ix, bestnr in enumerate(g_ifar['bestnr']):
-    g_ifar['stat'][ix] = (full_time_veto_max_bestnr > bestnr).sum()
-g_ifar['stat'] = g_ifar['stat'] / total_trials
-# 3) Missed due to vetoes (missed) 'g_missed2'
-g_missed2 = grab_triggers_subset(found_inj, missed, found_trig)
-
-# Statistics: missed-found
-MF = np.argsort(g_ifar['stat'])
-# Statistics: found-missed
-FM = MF[::-1]
-
-logging.info("%d found injections analysed.", len(found_injs))
+# 1) Found louder than background
+louder_mask = found_after_vetoes_stat > max_reweighted_snr
+louder_indices = found_after_vetoes[louder_mask]
+found_louder = grab_triggers_subset(inj_data, louder_indices)
+found_louder['reweighted_snr'] = f['found_after_vetoes/stat'][louder_mask]
+
+# 2) Found quieter than background
+quieter_mask = (found_after_vetoes_stat <= max_reweighted_snr) & (found_after_vetoes_stat != 0)
+quieter_indices = found_after_vetoes[quieter_mask]
+found_quieter = grab_triggers_subset(inj_data, quieter_indices)
+found_quieter['reweighted_snr'] = f['found_after_vetoes/stat'][quieter_mask]
+
+# Extract inclusive/exclusive IFAR for injections found quieter than background
+if opts.far_type == 'inclusive':
+    found_quieter['ifar'] = f['found_after_vetoes/ifar'][quieter_mask]
+elif opts.far_type == 'exclusive':
+    found_quieter['ifar'] = f['found_after_vetoes/ifar_exc'][quieter_mask]
```

Comment:
`found_quieter['ifar'] = f['found_after_vetoes/ifar'][quieter_mask] if opts.far_type == 'inclusive' else f['found_after_vetoes/ifar_exc'][quieter_mask]`

or (perhaps easier to read)
```
ifar_string = 'found_after_vetoes/ifar' if opts.far_type == 'inclusive' else 'found_after_vetoes/ifar_exc'
found_quieter['ifar'] = f[ifar_string][quieter_mask]
```